gpumon/jbuild

```diff
@@ -38,7 +37,7 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
     threads
     xcp))
   (wrapped false)
-  %s))
```

Comment:
Please keep the `coverage_rewriters` otherwise this would break @edwintorok coverage work

Reply:
So, keep `%s` and `coverage_rewriter` in lines 41 and 56? Or just line 56?

Reply:
Both